core/commands/files.go

```diff
@@ -997,26 +997,33 @@ Remove files or directories.
 			path = path[:len(path)-1]
 		}
 
+		// if '--force' specified, it will remove anything else,
+		// including file, directory, corrupted node, etc
+		force, _ := req.Options[forceOptionName].(bool)
+
 		dir, name := gopath.Split(path)
-		parent, err := mfs.Lookup(nd.FilesRoot, dir)
+
+		pdir, err := getParentDir(nd.FilesRoot, dir)
 		if err != nil {
+			if force {
```

Comment:
```suggestion
			if force && err == os.ErrNotExist {
```

Reply:
Just to be clear: here I'm suggesting using an if statement _instead_ of the switch

Reply:
yep, good point. Done!